apps/files_reminders/src/components/SetCustomReminderModal.vue

```diff
@@ -93,16 +93,16 @@ export default Vue.extend({
 	},
 
 	computed: {
-		fileId(): number {
-			return this.node.fileid
+		fileId(): number|undefined {
+			return this.node?.fileid
 		},
 
-		fileName(): string {
-			return this.node.basename
+		fileName(): string|undefined {
+			return this.node?.basename
 		},
 
 		name() {
-			return t('files_reminders', 'Set reminder for "{fileName}"', { fileName: this.fileName })
+			return this.node ? t('files_reminders', 'Set reminder for "{fileName}"', { fileName: this.fileName }) : ''
```

Comment:
```suggestion
			return this.filename ? t('files_reminders', 'Set reminder for "{fileName}"', { fileName: this.fileName }) : ''
```
For less cognitive overhead ;)

Reply:
Yes, makes sense :)